### DISCUSSION TITLE
Conda install produces an "Unsolveable Error" with no other information

### DISCUSSION THREAD

**Discussion by Sriram2631** [idaholab/moose#19948] [Q&A Getting Started] [answered]

**Comment by Sriram2631** [comment]
Following the steps for installation, the conda install command produces an "Unsolveable Error" and provides no further information. Unsure whether this is an issue with conda or MOOSE or with my linux system itself. 
For reference: OS - Ubuntu 20.04.3 LTS

**Comment by loganharbour** [comment, marked as answer]
Converted to a discussion.

@Sriram2631 can you please provide more information? What steps did you do exactly? Show the commands and paste the entire error.

**Reply by GiudGiud** [reply]
@Sriram2631
